packages/wallets/alpha/src/adapter.ts

```diff
@@ -46,13 +46,12 @@ declare const window: AlphaWindow;
 
 export interface AlphaWalletAdapterConfig {}
 
-export const AlphaWalletName = 'Alpha' as WalletName<'Alpha'>;
+export const AlphaWalletName = 'Alpha Wallet' as WalletName<'Alpha Wallet'>;
```

Comment:
In general, we have avoided superfluous use of `Wallet` in wallet names unless the wallet predated Wallet Adapter itself.